imap_processing/cdf/config/imap_hit_l1a_variable_attrs.yaml

```diff
@@ -14,6 +14,24 @@ default_attrs: &default
   UNITS: ' '
   SCALETYP: linear
 
+default_uint8_attrs: &default_uint8
+  <<: *default
+  FILLVAL: 255
+  FORMAT: I3
+  VALIDMIN: 0
+  VALIDMAX: 255
+  dtype: uint8
+  DEPEND_0: sc_tick
```

Comment:
I have two time variables in this dataset. EPOCH contains time per science frame (which is a group of 20 packets and is how most of the data is organized) and SC_TICK contains time per packet for the CCSDS headers which have data per packet rather than science frame. I need to double check with the instrument team on what format they need the SC_TICK time to be in. Is there criteria for using two time variables in a dataset?

Reply:
The link you shared answers my question: _Different time resolution data can be supported in a single CDF data set by defining the variables Epoch, Epoch_1, Epoch_2, etc. each representing a different time resolution_